004_team_2_agent_design/004_team_2_agent_design.tex

```diff
@@ -1,71 +1,76 @@
 \chapter{Team 2 Agent Design}\label{team_2_agent_design}
+
+\SetKwComment{Comment}{$\triangleright$\ }{}
+
 \section{Core Idea and Overall Strategy}
-The objective of this agent design was to implement a form of reinforcement learning to allow for an increase in performance and thus utility over time. This must be met without compromising on the overall goal of the simulations, whereby the agent must strike a balance between prioritising individual utility or prioritising collective utility, the latter involving a level of cooperation with the other agent types. Certain parallels can be drawn to evolutionary ecology, as the Tower with the Resident agents can represent a closed ecosystem, with the Platform providing a limited food source. 
+The objective of this agent design was to implement a form of reinforcement learning to allow for an increase in performance and thus utility over time. This must be met without compromising on the overall goal of the simulations, whereby the agent must attempt to strike a balance between prioritising the micro-level goal of maximal individual utility or prioritising the macro-level goal of maximising collective utility through sustainable resource management, the latter involving a level of cooperation with the other agent types. Certain parallels can be drawn to evolutionary ecology, as the Tower with the Resident agents can represent a closed ecosystem, with the Platform providing a limited food source.
```

Comment:
Should 'platform' have its first letter capitalized?

Reply:
Same sentiment for 'tower'

Reply:
We should at least standardize this across the report

Reply:
Yes I will capitalise.